reference-implementation/index.js

```diff
@@ -2,14 +2,12 @@
 
 {
   const defineMath = (name, assignment) => {
-    var configurable = typeof assignment === "function" ? true : false;
-    var writable = typeof assignment === "function" ? true : false;
-    var enumerable = typeof assignment === "function" ? true : false;
+    const configurable = writable = enumerable = typeof assignment === "function";
```

Comment:
Do you understand that `writable` and `enumerable` leak to global?

Reply:
oh that was intended behaviour?

Reply:
ooooh, now I get it

Reply:
yeah it was the first time I used that syntax, didn't know that the variable modifier needs to be repeated for each variable

make sense

Reply:
The fact that it leaks to global with `const a = b = 5;`, but you can't do `const a = const b = 5;` in order to not leak is weird ngl.
Seems counter-intuitive.

ig
```js
let a, b;
a = b = 5;
```
would be the way to go, but wow